### PR TITLE
Update unit tests

### PR DESCRIPTION
**Pull request by naveentatikonda**
### Description
Add build scripts and update CI for unit-tests.

### Issues Resolved
https://github.com/opensearch-project/logstash-input-opensearch/issues/8

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .ci/unit-test/logstash-run.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -ex
+
+export PATH=$BUILD_DIR/gradle/bin:$PATH
+
+bundle exec rspec -fd spec/inputs -t ~integration -t ~secure_integration
```
Comment (martin-gaievski) [inline .ci/unit-test/logstash-run.sh:6]: nit: newline at the file's end

File: spec/inputs/integration/opensearch_spec.rb [hunk 2]
```diff
@@ -54,8 +54,8 @@
   end
 
   describe 'against a secured opensearch', :secure_integration => true do
-    let(:user) { ENV['ELASTIC_USER'] || 'simpleuser' }
-    let(:password) { ENV['ELASTIC_PASSWORD'] || 'abc123' }
+    let(:user) { 'admin' }
+    let(:password) { 'admin' }
```
Comment (martin-gaievski) [inline spec/inputs/integration/opensearch_spec.rb:58]: why we dropped ability to set user/password and left only hardcoded values?

Comment (naveentatikonda) [reply spec/inputs/integration/opensearch_spec.rb:58]: We most commonly use 'admin'...'admin'. But, it is good to have the ability to set them, will add it back.

File: .ci/unit-test/logstash-run.sh (hunk 1 above)
Comment (naveentatikonda) [reply .ci/unit-test/logstash-run.sh:6]: ack

**Review by martin-gaievski — APPROVED** [review]

**Review by Shivamdhar — APPROVED** [review]

**Pull request merged**